2020/08/06 17:55:46 [error] 27#27: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.22.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.22.0.3:9000", host: "localhost"
2020/08/06 17:55:46 [error] 27#27: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.22.0.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://172.22.0.3:9000", host: "localhost", referrer: "http://localhost/"
2020/08/06 17:58:00 [error] 27#27: *4 connect() failed (111: Connection refused) while connecting to upstream, client: 172.22.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.22.0.3:9000", host: "localhost"
2020/08/06 18:01:59 [error] 27#27: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.22.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.22.0.3:9000", host: "localhost"
2020/08/06 18:02:06 [error] 27#27: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.22.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.22.0.3:9000", host: "localhost"
2020/08/06 19:05:24 [error] 27#27: *4 connect() failed (111: Connection refused) while connecting to upstream, client: 172.22.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.22.0.3:9000", host: "localhost"
2020/08/06 19:05:30 [error] 27#27: *4 connect() failed (111: Connection refused) while connecting to upstream, client: 172.22.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.22.0.3:9000", host: "localhost"
2020/08/06 19:08:20 [error] 27#27: *7 connect() failed (111: Connection refused) while connecting to upstream, client: 172.22.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.22.0.3:9000", host: "localhost"
2020/08/06 19:09:24 [error] 27#27: *9 connect() failed (111: Connection refused) while connecting to upstream, client: 172.22.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.22.0.3:9000", host: "localhost"
2020/08/06 19:13:52 [error] 27#27: *11 connect() failed (111: Connection refused) while connecting to upstream, client: 172.22.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.22.0.3:9000", host: "localhost"
2020/08/06 19:15:21 [error] 27#27: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.22.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.22.0.3:9000", host: "localhost"
2020/08/06 19:19:21 [emerg] 39#39: bind() to 0.0.0.0:80 failed (98: Address already in use)
2020/08/06 19:19:21 [emerg] 39#39: bind() to 0.0.0.0:80 failed (98: Address already in use)
2020/08/06 19:19:21 [emerg] 39#39: bind() to 0.0.0.0:80 failed (98: Address already in use)
2020/08/06 19:19:21 [emerg] 39#39: bind() to 0.0.0.0:80 failed (98: Address already in use)
2020/08/06 19:19:21 [emerg] 39#39: bind() to 0.0.0.0:80 failed (98: Address already in use)
2020/08/06 19:19:21 [emerg] 39#39: still could not bind()
2020/08/06 19:24:25 [error] 28#28: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.22.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.22.0.3:9000", host: "localhost"
2020/08/06 19:26:31 [error] 28#28: *3 connect() failed (111: Connection refused) while connecting to upstream, client: 172.22.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.22.0.3:9000", host: "localhost"
2020/08/06 19:52:31 [error] 28#28: *5 FastCGI sent in stderr: "PHP message: [debug] Notified event "kernel.request" to listener "Symfony\Component\HttpKernel\EventListener\DebugHandlersListener::configure".
PHP message: [debug] Notified event "kernel.request" to listener "Symfony\Component\HttpKernel\EventListener\ValidateRequestListener::onKernelRequest".
PHP message: [debug] Notified event "kernel.request" to listener "Symfony\Component\HttpKernel\EventListener\SessionListener::onKernelRequest".
PHP message: [debug] Notified event "kernel.request" to listener "Symfony\Component\HttpKernel\EventListener\LocaleListener::setDefaultLocale".
PHP message: [debug] Notified event "kernel.request" to listener "Symfony\Component\HttpKernel\EventListener\RouterListener::onKernelRequest".
PHP message: [error] Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /"" at /var/www/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136
PHP message: [debug] Notified event "kernel.exception" to listener "Symfony\Component\HttpKernel\EventListener\ErrorListener::logKernelException".
PHP message: [debug] Notified event "kernel.exception" to listener "Symfony\Component\HttpKernel\EventListener\RouterListener::onKernelException".
PHP message: [debug] Listener "Symfony\Component\HttpKernel\EventListener\RouterListener::onKernelException" stopped propagation of the event "kernel.exception".
PHP message: [debug] Listener "Symfony\Component\HttpKernel\EventListener\ErrorListener::onKernelException" was not called for event "kernel.exception".
PHP message: [debug] Notified event "kernel.response" to listener "Symfony\Component\HttpKernel\EventListener\ResponseListener::onKernelResponse".
PHP message: [debug] Notified event "kernel.response" to listener "Symfony\Component\HttpKernel\EventListener\ErrorListener::removeCspHeader".
PHP message: [debug] Notified event "kernel.response" to listener "Symfony\Component\HttpKernel\EventListener\DisallowRobotsIndexingListener::
2020/08/06 21:10:13 [crit] 29#29: *1 realpath() "/var/www/public" failed (2: No such file or directory), client: 172.22.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2020/08/06 21:10:13 [error] 29#29: *1 FastCGI sent in stderr: "Primary script unknown" while reading response header from upstream, client: 172.22.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.22.0.3:9000", host: "localhost"
2020/08/06 21:11:34 [crit] 27#27: *1 realpath() "/var/www/public" failed (2: No such file or directory), client: 172.22.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2020/08/06 21:11:34 [error] 27#27: *1 FastCGI sent in stderr: "Primary script unknown" while reading response header from upstream, client: 172.22.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.22.0.3:9000", host: "localhost"
2020/08/06 21:11:36 [crit] 27#27: *1 realpath() "/var/www/public" failed (2: No such file or directory), client: 172.22.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2020/08/06 21:11:36 [error] 27#27: *1 FastCGI sent in stderr: "Primary script unknown" while reading response header from upstream, client: 172.22.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.22.0.3:9000", host: "localhost"
2020/08/06 21:12:35 [crit] 27#27: *1 realpath() "/var/www/public" failed (2: No such file or directory), client: 172.22.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2020/08/06 21:12:35 [error] 27#27: *1 FastCGI sent in stderr: "Primary script unknown" while reading response header from upstream, client: 172.22.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.22.0.3:9000", host: "localhost"
2020/08/09 20:51:01 [warn] 28#28: *34 an upstream response is buffered to a temporary file /var/cache/nginx/fastcgi_temp/1/00/0000000001 while reading upstream, client: 172.22.0.1, server: localhost, request: "POST /transfer?amount=1&source_wallet_id=15&dest_wallet_id=16 HTTP/1.1", upstream: "fastcgi://172.22.0.3:9000", host: "localhost"
2020/08/09 20:53:24 [warn] 28#28: *36 an upstream response is buffered to a temporary file /var/cache/nginx/fastcgi_temp/2/00/0000000002 while reading upstream, client: 172.22.0.1, server: localhost, request: "POST /transfer HTTP/1.1", upstream: "fastcgi://172.22.0.3:9000", host: "localhost"
